Projects/Claims/Sources/Views/SubmitClaim/FilesUpload/FilesUploadScreen.swift

```diff
@@ -56,7 +56,8 @@ struct SubmitClaimFilesUploadScreen: View {
 
                             }
                             .fixedSize(horizontal: false, vertical: true)
-                            .clipShape(Squircle.default())
+                            .background(hSurfaceColor.Opaque.primary)
```

Comment:
Do we want to set background here? It wasn't here before

Reply:
not sure, I don't think it matters but I can remove it